test/cli/template-use/common-dir/out/project/.keboola/manifest.json

```diff
@@ -34,7 +34,7 @@
       "id": 12345,
       "path": "main",
       "metadata": {
-        "KBC.KAC.templates.instances": "[{\"instanceId\":\"%s\",\"instanceName\":\"My Instance\",\"templateId\":\"my-template-id\",\"repositoryName\":\"keboola\",\"version\":\"1.0.0\",\"created\":{\"date\":\"%s\",\"tokenId\":\"%s\"},\"updated\":{\"date\":\"%s\",\"tokenId\":\"%s\"}}]"
+        "KBC.KAC.templates.instances": "[{\"instanceId\":\"%s\",\"instanceName\":\"My Instance\",\"templateId\":\"my-template-id\",\"repositoryName\":\"keboola\",\"version\":\"1.2.3\",\"created\":{\"date\":\"%s\",\"tokenId\":\"%s\"},\"updated\":{\"date\":\"%s\",\"tokenId\":\"%s\"}}]"
```

Comment:
Tu bol bug, do metadat sa ukladala verzia, ktoru zadal uzivatel, a nie ta co sa nasla.
Bol to problem iba v CLI, kde sa nemusi zadat cela verzia, .... UI posiela vzdy celu verziu, takze tento problem sa neprejavil.